backend/sass/common.blocks/_wallet.scss

```diff
@@ -83,3 +83,27 @@
 .wallet__table-button--hamburger {
   padding: 0 15px;
 }
+
+.button.wallet__copy-signed {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+
+  background: $chainweaver-blue;
+  color: white;
+
+  .copy-status {
+    transition: opacity, transform 0.25s linear;
+  }
+
+  .copy-success {
```

Comment:
Should this include the `&` : `&.copy-success` ?

Reply:
Hmm not like this, but it should if they're nested inside `.copy-status` like the setup ones are.
I'll nest here as well